core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3194,6 +3194,7 @@ paths:
       parameters:
         - $ref: '#/components/parameters/workspaceNamespacePathParam'
         - $ref: '#/components/parameters/workspaceNamePathParam'
+        - $ref: '#/components/parameters/userProjectQueryParam'
```

Comment:
as experienced in https://broadworkbench.atlassian.net/browse/CORE-95, this addition will be a breaking change for any generated Rawls clients. I believe some of them are out there, even if they are not supported products. Is this ok?